charts/warpstream-agent/values.yaml

```diff
@@ -63,6 +63,12 @@ affinity: {}
 
 priorityClassName: ""
 
+# Optional disruption budget
+pdb:
+  create: false
+  minAvailable: 1
```

Comment:
would it be possible to add some comments before each field? Something along the lines of what is described here https://kubernetes.io/docs/tasks/run-application/configure-pdb/ to let the user know that it can be an integer or a percentage?

Maybe we should put some default values as well? Say 75% for minAvailable and 33% for maxUnavailable?

Reply:
yes we can add a few comments. I sort of assumed that the users specifying a pdb know what they are doing. Good idea to pick good default values for the warpstream agent. This is probably some kind of info you also want to add on this guide: https://docs.warpstream.com/warpstream/reference/infrastucture-as-code/helm-charts. 

btw feel free to commit on top of my PR

Reply:
unless you have users that deploy many (> 5) warpstream-agent pod replicas, I find integer values easier to understand (you don't need to care about the rounding logic).